python/src/opendp/_convert.py

```diff
@@ -420,6 +420,9 @@ def _vector_to_slice(val: Sequence[Any], type_name: RuntimeType) -> FfiSlicePtr:
     if np is not None and isinstance(val, np.ndarray):
         val = val.tolist()
 
+    if isinstance(val, tuple):
+        val = list(val)
+
     if not isinstance(val, list):
         raise TypeError(f"Expected type is {type_name} but input data is not a list.")
```

Comment:
Maybe just:
```
else:
    val = list(val)
```
Apart from strings, if `list()` works, it can probably be treated as a `list`, unless this is a case where we want to be paranoid?